deploy/roles/apache/tasks/main.yml

```diff
@@ -51,3 +48,7 @@
 - name: Save rules
   command: "/sbin/service iptables save"
   when: iptables_config
+
+- name: create certs for apache localhost
+  command: openssl req -x509 -nodes -days 365 -newkey rsa:2048 -subj "/C=US/ST=CFPB/L=Washington/O=Dis/CN=www.cf.gov" -keyout /etc/pki/tls/private/localhost.key -out /etc/pki/tls/certs/localhost.crt
+  when: not custom_repo
```

Comment:
I would add `creates: /etc/pki/tls/certs/localhost.crt`, this ensures it does not recreate the cert if it already exists.